mappings/net/minecraft/block/VineLogic.mapping

```diff
@@ -1,3 +1,4 @@
 CLASS net/minecraft/class_4949 net/minecraft/block/VineLogic
 	METHOD method_25961 isValidForWeepingStem (Lnet/minecraft/class_2680;)Z
 		ARG 0 state
+	METHOD method_26381 getGrowthLength (Ljava/util/Random;)I
```

Comment:
The name for the method itself is good, but make sure to map the parameter to `random` as well.